test/e2e/framework/psp_util.go

```diff
@@ -75,6 +77,61 @@ func PrivilegedPSP(name string) *extensionsv1beta1.PodSecurityPolicy {
 	}
 }
 
+// RestrictedPSP creates a PodSecurityPolicy that is most strict.
+func RestrictedPSP(name string) *extensionsv1beta1.PodSecurityPolicy {
```

Comment:
same here, why not keep this private to the PSP test?

Reply:
To reduce a chances to introduce a duplicated function in other tests in future.